core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -24,34 +30,78 @@
-  public static final MetricName RPC_SERVER_REQUEST_BYTES =
-      new MetricName("/rpc/server/request_bytes");
-  public static final MetricName RPC_SERVER_RESPONSE_BYTES =
-      new MetricName("/rpc/server/response_bytes");
-  public static final MetricName RPC_SERVER_SERVER_LATENCY =
-      new MetricName("/rpc/server/server_latency");
-  public static final MetricName RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES =
-      new MetricName("/rpc/server/uncompressed_request_bytes");
-  public static final MetricName RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES =
-      new MetricName("/rpc/server/uncompressed_response_bytes");
-
-  //Visible for testing
+  public static final MeasurementDescriptor RPC_CLIENT_ERROR_COUNT =
+      new MeasurementDescriptor(
+          "/rpc/client/error_count",
+          "RPC Errors",
+          new MeasurementUnit(1, scalar));
+  public static final MeasurementDescriptor RPC_CLIENT_REQUEST_BYTES =
+      new MeasurementDescriptor(
+          "/rpc/client/request_bytes",
+          "Request MB/s",
+          new MeasurementUnit(6, bytes, secs));
+  public static final MeasurementDescriptor RPC_CLIENT_RESPONSE_BYTES =
+      new MeasurementDescriptor(
+          "/rpc/client/response_bytes",
+          "Response MB/s",
+          new MeasurementUnit(6, bytes, secs));
+  // TODO(dpo): verify this encoding
+  public static final MeasurementDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY =
+      new MeasurementDescriptor(
+          "/rpc/client/roundtrip_latency",
+          "RPC roundtrip latency us",
+          new MeasurementUnit(1, scalar));
+  public static final MeasurementDescriptor RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES =
+      new MeasurementDescriptor(
+          "/rpc/client/uncompressed_request_bytes",
+          "Uncompressed Request MB/s",
+          new MeasurementUnit(6, bytes, secs));
+  public static final MeasurementDescriptor RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES =
+      new MeasurementDescriptor(
+          "/rpc/client/uncompressed_response_bytes",
+          "Uncompressed Request MB/s",
+          new MeasurementUnit(6, bytes, secs));
+
+  public static final MeasurementDescriptor RPC_SERVER_ERROR_COUNT =
+      new MeasurementDescriptor(
+          "/rpc/server/error_count",
+          "RPC Errors",
+          new MeasurementUnit(1, scalar));
+  public static final MeasurementDescriptor RPC_SERVER_REQUEST_BYTES =
+      new MeasurementDescriptor(
+          "/rpc/server/request_bytes",
+          "Request MB/s",
+          new MeasurementUnit(6, bytes, secs));
+  public static final MeasurementDescriptor RPC_SERVER_RESPONSE_BYTES =
+      new MeasurementDescriptor(
+          "/rpc/server/response_bytes",
+          "Response MB/s",
+          new MeasurementUnit(6, bytes, secs));
+  public static final MeasurementDescriptor RPC_SERVER_SERVER_LATENCY =
+      new MeasurementDescriptor(
+          "/rpc/server/server_latency",
+          "Latency in msecs",
+          new MeasurementUnit(3, scalar));
```

Comment:
Agreed - I'm following the specifications in resource_definitions.txt but I'll verify the units with Kun.